packages/@divvi/mobile/src/tokens/selectors.test.ts

```diff
@@ -159,6 +160,15 @@ const state: any = {
         isNative: true,
         minimumAppVersionToSwap: '1.10.0',
       },
+      ['unsupported-network:0x8']: {
```

Comment:
i think that it is implied in a few tests that tokens on unsupported networks are omitted from the selector results, maybe it's helpful to add a comment in those tests so that it is more explicit where this mock data is used? i also wonder whether this scenario is real, because tokens are fetched by network id on app start so in theory there should be no tokens cached that don't match the specified supported networks

Reply:
left a comment. and you're right this isn't a valid scenario, but since the tokensById selector does this filtering as an extra guardrail, I kept this. I am open to drop the additional filtering too.